### PR TITLE
Tailwindcss

### PR DESCRIPTION
**Pull request by KiveshVirasamy**
Please see where I am going wrong this is not working its supposed too.

### CODE REVIEW

File: tailwind.config.cjs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{html,js,ts}"],
```
Comment (CharelMarais) [inline tailwind.config.cjs:3]: content: ["./src/**/*.{html,js,ts}", "./index.html" ] to allow tailwind to use the tailwind in your HTML file

File: index.html [hunk 2]
```diff
@@ -1,45 +1,44 @@
...
 
 <head>
-  <!-- The head section contains metadata and external links to CSS and JavaScript files. -->
   <meta charset="UTF-8">
   <title>Flight Tracker</title>
   <link rel="icon" type="image/svg+xml" href="./assets/img/icons8-plane-24.png">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./src/styles.scss">
+  <!-- Tailwind CSS -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/2.2.17/tailwind.min.css">
```
Comment (CharelMarais) [inline index.html:10]: You do not need line 10 if you adjust the tailwind config file.

**Review by CharelMarais — CHANGES_REQUESTED** [review]
Please check the comments on the needed adjustments.

**Comment by JonathanLadwig** [comment]
I don't see any problems. You're good to merge.

**Review by JonathanLadwig — APPROVED** [review]
Rest Easy My Son

**Comment by JonathanLadwig** [comment]
Just fix your merge conflicts

**Pull request merged**